frontend/.gitignore

```diff
@@ -22,3 +22,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+src/pages/__snapshots__
```

Comment:
I thought we wanted snapshots in source?

Reply:
yeah, we want them in git, please revert this line